docs/index.rst

```diff
@@ -14,57 +15,40 @@ Django GUID
 .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
     :target: https://django-guid.readthedocs.io/en/latest/?badge=latest
 
-Django GUID stores a GUID to an object, making it accessible by using the ID of the current thread.
-The GUID is accessible from anywhere within the application throughout a request,
-allowing us to inject it into the logs.
 
-* Free software: BSD License
-* Homepage: https://github.com/JonasKs/django-guid
-* Documentation: https://django-guid.readthedocs.io
-
-
-Example
--------
+Django GUID attaches a unique correlation ID to all your log outputs for every requests you handle. This means that every error now has an ID connecting it to all other relevant logs, making
+debugging simple.
 
-Using ``demoproj`` as an example, all the log messages **without** ``django-guid`` would look like this:
+Integration are also made possible, as the ID can be *returned as a header*,
+or can be added as an *outgoing header* in external requests, making it possible to extend the reach of correlation IDs to whole systems.
```

Comment:
Can be returned as a header, or added as an outgoing header? :thinking:

Reply:
Yes I think we need to rephrase some of these lines. The idea here was that you can return the ID to your backends, e.g., for error messages, *or* you send the ID in a request to another backend when processing a request.

Reply:
Changed it to "Integration are also made possible, as the ID can be returned as a header, or forwarded, making it possible to extend the reach of correlation IDs to whole systems." but feel free to jump in and make edits.